app/frontend/src/components/apiAccess/Step3.vue

```diff
@@ -1,24 +1,183 @@
 <template>
   <v-container>
-    Contents to go here
-    <br />
-    <br />
-    <br />
-    <br />
-    <div class="text-center">
-      <v-btn class="BC-Gov-PrimaryButton light mr-4" text @click="setStep(2)">Back</v-btn>
-      <v-btn class="BC-Gov-PrimaryButton" text @click="setStep(4)">Next</v-btn>
-    </div>
+    <h2 class="green--text text--darken-2 text-center mb-12">
+      <v-icon large color="green darken-2">check_circle</v-icon>API Access Granted
+    </h2>
+    <p>
+      Your service client for
+      <strong>{{ acronym }}</strong> has been successfully updated in the
+      <strong>{{ environment }}</strong>
+      Keycloak realm {{generatedClient}}
+    </p>
+
+    <v-form ref="form" lazy-validation>
+      <v-row>
+        <v-col cols="8">
+          <label>Service Client</label>
+          <v-text-field
+            v-model="generatedClient"
+            dense
+            hide-details="auto"
+            outlined
+            flat
+            readonly
+            solo
+          />
+        </v-col>
+      </v-row>
+
+      <v-row class="mt-2">
+        <v-col cols="8">
+          <label>Password</label>
+          <v-text-field
+            v-model="passwordShown"
+            dense
+            hide-details="auto"
+            outlined
+            flat
+            readonly
+            solo
+          />
+        </v-col>
+        <v-col cols="1" class="pt-9">
+          <v-tooltip bottom>
+            <template v-slot:activator="{ on }">
+              <v-btn v-on="on" text large icon class="BC-Gov-IconButton" @click="decryptPassword">
+                <v-icon>remove_red_eye</v-icon>
+              </v-btn>
+            </template>
+            <span>Decrypt and show Password</span>
+          </v-tooltip>
+        </v-col>
+        <v-col cols="1" class="pt-9 pl-6">
+          <v-tooltip bottom>
+            <template v-slot:activator="{ on }">
+              <v-btn
+                v-on="on"
+                text
+                large
+                icon
+                class="BC-Gov-IconButton"
+                v-clipboard:copy="passwordShown"
```

Comment:
Have we talked to Mina about including the token endpoint?  Or downloading a config file that has service client/password/token url?  I think the token url is very important to incorporate here somewhere/somehow.

Reply:
Yeah I think I remember some discussion about a download option.
Funny, Marco was asking me about the prod token URL just yesterday and where he could find it, so probably a good idea to make it very forward.
It is in the POSTMAN collection, but that's only for the 1 environment the collection is for.

Will track in https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-922